x-pack/solutions/security/plugins/security_solution_serverless/public/navigation/util.ts

```diff
@@ -24,21 +33,21 @@ export const getProjectFeaturesUrl = (cloud: CloudStart): string | undefined =>
 };
 
 export const getCloudUrl: GetCloudUrl = (cloudUrlKey, cloud) => {
+  const cloudUrls = cloud.getUrls();
+
   switch (cloudUrlKey) {
-    case 'billing':
-      return cloud.billingUrl;
```

Comment:
`billing` is not needed here, which is good because that would have forced this function to be asynchronous